crates/biome_css_analyze/src/keywords.rs

```diff
@@ -5433,17 +5433,293 @@ pub const RESET_TO_INITIAL_PROPERTIES_BY_FONT: [&str; 13] = [
     "font-variation-settings",
 ];
 
+// https://developer.mozilla.org/ja/docs/Web/HTML/Element
+// https://github.com/sindresorhus/html-tags/blob/main/html-tags.json
+pub(crate) const HTML_TAGS: [&str; 148] = [
+    "a",
+    "abbr",
+    "acronym",
+    "address",
+    "applet",
+    "area",
+    "article",
+    "aside",
+    "audio",
+    "b",
+    "base",
+    "basefont",
+    "bdi",
+    "bdo",
+    "bgsound",
+    "big",
+    "blink",
+    "blockquote",
+    "body",
+    "br",
+    "button",
+    "canvas",
+    "caption",
+    "center",
+    "cite",
+    "code",
+    "col",
+    "colgroup",
+    "content",
+    "data",
+    "datalist",
+    "dd",
+    "del",
+    "details",
+    "dfn",
+    "dir",
```

Comment:
`dialog` is missing

ref https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog